admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -5,6 +5,63 @@ Reverse proxy configuration
 Nextcloud can be run through a reverse proxy, which can cache static assets such
 as images, CSS or JS files, move the load of handling HTTPS to a different
 server or load balance between multiple servers.
+In this example, we will be assuming that we are using NGiNX and Certbot.
+
+Setting up Reverse Proxy
+------------------------
+
+All requests will initially be received by the reverse proxy which will then
+perform SSL termination and route the request to the correct server. SSL
+certificate is managed by Certbot.
+
+We will need to configure the reverse proxy to handle the requests. So we will
+utilize the following configuration on the reverse proxy server:
+
+::
+
+ server {
+    listen 80;
+	  server_name nextcloud.domain.com;
+
+	return 301 https://$server_name$request_uri;
+  location /.well-known {
+    root /var/www/nextcloud/;
+    allow all;
+  }
+  }
+
+ server {
+	listen 443 ssl;
+	server_name nextcloud.domain.com;
+
+	access_log /var/log/nginx/nextcloud.domain.com.access.log;
+	error_log /var/log/nginx/nextcloud.domain.com.error.log;
+
+	client_max_body_size 0;
+	underscores_in_headers on;
+
+	ssl on;
+        ssl_certificate /etc/letsencrypt/live/nextcloud.domain.com/fullchain.pem; # managed by Certbot
+        ssl_certificate_key /etc/letsencrypt/live/nextcloud.domain.com/privkey.pem; # managed by Certbot
+        include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
+        ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
+
+	ssl_stapling on;
+	ssl_stapling_verify on;
```

Comment:
The SSL stuff can be dropped completely and shouldn't be part of our documentation as Nextcloud doesn't require a specific config.